packages/@coorpacademy-components/src/atom/select/style.css

```diff
@@ -132,6 +135,22 @@
   z-index: 11;
 }
 
+.selectedArrow {
+  composes: arrow;
+}
+
+.player .selectWrapper .arrow {
+  color: cm_grey_450;
+}
+
+.player:hover .arrow {
+  color: cm_blue_900;
+}
+
+.player:hover .selectedArrow {
+  color: inherit;
```

Comment:
`color: inherit` to use the parents color (skin) on selection, without having to handle states